bash_tests/README.md

```diff
@@ -0,0 +1,67 @@
+## Bash Scripts to test Watson AI Services
+
+This folder contains bash scripts that are using **curl** command to call Watson APIs as explained on Watson getting started pages.
+
+Test whether **curl** is installed. Run the following command on the command line. 
+
+```bash
+curl -V
+```
+
+If the output lists the curl version with SSL support, you are all set otherwise install it.
+
+```bash
+sudo apt install curl
+```
+To run a script, 
+* change to the folder where you have the scripts:
+```bash
+cd ~/TJBOT/bash_tests
+```
+* type **./** followed by the name of script file then space then your api-key for the related service
+
+```bash
+./tone_analyzer_4.sh pomy6gsgO4kmjtNa3eiuhg7haziujhyf0Ps8WxTrIom6
```

Comment:
is this an actual Watson credential? it should be revoked if it is, not wise to commit to git. we may want to replace this with some placeholder text, e.g. `<insert Watson key here>`

Reply:
No this is a fake one just to make it clear for users. I will replace it with `<insert Watson key here>` as suggested.